ngraph/core/src/pass/manager.cpp

```diff
@@ -34,6 +36,47 @@
 using namespace std;
 using namespace ngraph;
 
+namespace ngraph
+{
+    namespace pass
+    {
+        namespace
+        {
+            class PerfCounters
```

Comment:
Why do we need this class inside the nGraph?

Only for `run_passes`?

Reply:
Yes, it is required only for run_passes and should be hidden in public API.